saleor/graphql/views.py

```diff
@@ -46,8 +47,10 @@ def __init__(
     def dispatch(self, request, *args, **kwargs):
         # Handle options method the GraphQlView restricts it.
         if request.method == 'GET':
-            return render_to_response('graphql/playground.html')
-        if request.method == 'OPTIONS':
+            if settings.DEBUG:
+                return render_to_response('graphql/playground.html')
+            response = self.handle_query(request, *args, **kwargs)
```

Comment:
Shouldn't we throw a `405 - method not allowed`, instead?

Reply:
You mean ```405``` should be raised when it is not in DEBUG mode?

Reply:
Yup!

Reply:
Sure, if Saleor has no plan for support GET method of GraphQL, it is reasonable to do that.